INSTALLATION.md

```diff
@@ -0,0 +1,156 @@
+
+Now we need to install the required packages on all the nodes, the engine as well as the VM hosts:
+```
+./15-install-packages.sh http://bob.eng.lab.tlv.redhat.com/builds/4.5/rhv-4.5.3-1/api/ 
+[LAB] Install engine
+...
+[LAB] Install hosts
+...
+[LAB] oVirt packages done
+```
+
+No longer needed:
+~~**Make sure to reboot the nodes after this step!**~~
+
+```bash
+for i in $(cat engine) $(cat hosts); do ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -o User=root $i reboot; done
+```
+
+
+The next step takes care of network configuration, dnsmasq for RHV and the SD data share. If you want to set a cluster name different to the default of *perfscale*, please do so in the **common_funcs** file. Also make sure the domain is set correctly. This information relates to the name and domain you set in your OCP cluster.
+
+```bash
+./20-prepare-engine-host.sh
+[LAB] Engine networking
+Connection 'Wired connection 1' (71dece5b-b869-3b20-90a9-b1393dabf91e) successfully deleted.
+Connection 'Wired connection 2' (b60e47bd-bcb1-366d-94c7-c6ac6570a1c2) successfully deleted.
+Connection 'Wired connection 3' (7315ca38-fb52-39ad-80aa-2339c22dcb39) successfully deleted.
+Connection 'baremetal' (6c139a32-acf9-4b9b-82fe-9cc77a42ada5) successfully added.
+Connection 'ens2f0' (78ca32a3-da33-4784-b3e8-5727d12faf95) successfully added.
+Connection 'provisioning' (ece02a74-a81c-4a35-adc2-332c1c94a143) successfully added.
+Connection 'ens2f1' (1e177e0d-24aa-4791-a2c0-4a676bc755a1) successfully added.
+Connection successfully activated (master waiting for slaves) (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/9)
+Connection successfully activated (master waiting for slaves) (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/11)
+Connection successfully activated (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/13)
+success
+success
+success
+success
+Warning: ALREADY_ENABLED: masquerade
+success
+success
+success
+[LAB] Setup dnsmasq
+dnsmasq.conf                                                                                                                                100%  300     2.2KB/s   00:00
+dnsmasq: syntax check OK.
+Created symlink /etc/systemd/system/multi-user.target.wants/dnsmasq.service → /usr/lib/systemd/system/dnsmasq.service.
+[LAB] Create data SD share
+exporting *:/srv/data
+Created symlink /etc/systemd/system/multi-user.target.wants/nfs-server.service → /usr/lib/systemd/system/nfs-server.service.
+```
+
+Let's get the engine set up:
+```bash
+./25-setup-ovirt-engine.sh
+[LAB] Run engine-setup
+...
+```
+
+Plus the storage:
+```bash
+./30-setup-host-storage.sh
+[LAB] Create hook to pin VMs
+...
+  Logical volume "pool0" created.
+...
+[LAB] Add data SD
+...
+[LAB] Attach data SD to Default DC
+...
+[LAB] Wait for DC to be up
+...
+```
+
+Add the hypervisor hosts. Make sure you don't have any duplicates in the hosts file!
+```bash
+./35-add-ovirt-hosts.sh
```

Comment:
maybe note that if the nics are not interconnected this will never finish. It may be easy enough to log into RHV and troubleshoot from there.